tests/test_mcts.py

```diff
@@ -124,8 +124,8 @@ def test_backup_incorporate_results(self):
         self.assertEqual(leaf.N, 2)
         self.assertEqual(leaf2.N, 1)
         # average of 0, -1, -0.2
+        self.assertAlmostEqual(leaf.Q, root.child_Q[leaf.fmove])
```

Comment:
I had left all these tests in when doing my child_X refactor just to double check I didn't break anything, but we could probably nuke all these redundant tests now.

Reply:
i don't think there's anything hurt by having them, and actually they were pretty helpful for me to look at when reviewing the vlosses code.  I'm happy with them.